themes/codefor-theme/layouts/blog/list.html

```diff
@@ -13,8 +13,9 @@
                 {{ partial "blog-preview.html" . }}
             {{ end }}
 
+            <!-- For Twitter we add the hidden class which will be removed in script below -->
             {{if eq $paginator.PageNumber 1}}
-            <div class="col-24 col-md-12 mb-5 mb-md-7 project-preview-container twitter-container" >
+            <div class="col-24 col-md-12 mb-5 mb-md-7 project-preview-container twitter-container hidden" >
```

Comment:
Bootstrap already contains such a helper class (`d-none`)

```suggestion
            <div class="col-24 col-md-12 mb-5 mb-md-7 project-preview-container twitter-container d-none" >
```